sdk/jobs/automl-standalone-jobs/automl-image-classification-multilabel-task-fridge-items/automl-image-classification-multilabel-task-fridge-items.ipynb

```diff
@@ -204,6 +286,44 @@
     "# my_validation_data_input = Input(type=AssetTypes.MLTABLE, path=\"azureml://datastores/workspaceblobstore/paths/vision-classification/valid\")"
```

Comment:
not related to this change.. but maybe we can add another commented line here on how to use MLTable with V1 datasets?

my_training_data_input = Input(
    type=AssetTypes.MLTABLE, path="azureml:azureml_mighty_rod_ql9bz1gzcp_input_data_TrainData:1",
    mode=InputOutputModes.DIRECT)